src/main/java/com/thealgorithms/backtracking/KnightsTour.java

```diff
@@ -13,7 +13,7 @@
     Input : N = 8
 
     Output:
-        0  59  38  33  30  17   8  63
+        0   59  38  33  30  17   8  63
```

Comment:
```suggestion
        0  59  38  33  30  17  8  63
```
Remove the extra space